assets/css/styles.css

```diff
@@ -721,14 +738,291 @@ HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
   }
 }
 
+/*
+  ADD PROJECT POPUP CSS
+  - Add the Popup class (same as the html file and style it)
+  --  Add .popup button and .popup h1, h2, h3 and p to respective
+      class declarations in the different sections
+  - Add the class open-popup (to be added in the javascript open function)
+*/
+.modal-container {
+  width: 100vw;
+  height: 100vh;
+  position: relative;
+  top: 0;
+}
+
+.popup-modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  background: var(--my-competences-boxes-color);
+  z-index: 1;
+  filter: opacity(95%);
+  overflow-y: hidden;
+  overscroll-behavior: none;
+}
+
+.project-popup {
+  width: 100%;
+  position: absolute;
+  background: var(--image-placeholder-color1);
+  border-radius: 1%;
+  top: 0;
+  left: 0;
+  text-align: center;
+  padding: 0;
+  transition: transform 0.4s, visibility 0.4s, top 0.4s;
+  visibility: visible;
+  z-index: 100;
+  backdrop-filter: blur(15px);
+  backdrop-filter: brightness(60%);
+  backdrop-filter: contrast(40%);
+}
+
+.js-project-popup-img {
+  width: 100%;
+  border-bottom: 0;
+}
+
+.project-popup-desktop-img {
+  width: 100%;
+  height: 50%;
+  margin-top: 3%;
+  border-radius: 1%;
+  border: 1px solid white;
+}
+
+.project-popup-mobile-img {
+  width: 100%;
+  height: 40% !important;
+  margin-top: 3%;
+  border-radius: 1%;
+  border: 1px solid white;
+}
+
+/*
+.project-popup-desktop-img img,
+.project-popup-mobile-img img {
+  width: 100% !important;
+  height: 100% !important;
+}
+*/
```

Comment:
-[ ] As requested by the previous reviewer please make sure to remove this commented code as it makes does not make the clean project.

Reply:
Done @roshan-bajgain